CHANGELOG

```diff
@@ -23,6 +23,12 @@ Miscellaneous
 - In SLISolver, changed the residual inner product from (Br,r) to (Br,Br) so the
   solver can work with non-SPD preconditioner B.
 
+- Added support for ADIOS2 for parallel I/O with ParaView visualization. 
+  The classes adios2stream and ADIOS2DataCollection are introduced in mfem as 
+  the interfaces to generate ADIOS2 Binary Pack (BP4) directory datasets for the 
+  entire spatial and temporal data. In addition, ADIOS2 allows for setting a 
+  user-defined number of data substreams/subfiles. 
+  For existing examples see config/run_adios2stream.py.
 
```

Comment:
Can you merge master in this branch and move the above to the mfem-4.1.1 section?